Source/Mosa.Packages/Mosa.Compiler.x86.nuspec

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0"?>
```

Comment:
undo the renaming of this file as this is not related to the Mosa.Platform.* projects
this file and nuget package name should stay as Mosa.Platform.x86